checker/.classpath

```diff
@@ -1,41 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="src" output="bin/main" path="src/main/java">
-		<attributes>
-			<attribute name="gradle_scope" value="main"/>
-			<attribute name="gradle_used_by_scope" value="main,test"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" output="bin/test" path="src/test/java">
-		<attributes>
-			<attribute name="gradle_scope" value="test"/>
-			<attribute name="gradle_used_by_scope" value="test"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" output="bin/test" path="src/test/resources">
-		<attributes>
-			<attribute name="gradle_scope" value="test"/>
-			<attribute name="gradle_used_by_scope" value="test"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" output="bin/testannotations" path="src/testannotations/java">
-		<attributes>
-			<attribute name="gradle_scope" value="testannotations"/>
-			<attribute name="gradle_used_by_scope" value="test,testannotations"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" output="bin/testannotations" path="src/testannotations/resources">
-		<attributes>
-			<attribute name="gradle_scope" value="testannotations"/>
-			<attribute name="gradle_used_by_scope" value="test,testannotations"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
-	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
-	<classpathentry kind="src" path="/framework"/>
+	<classpathentry kind="src" output="build/classes/java/main" path="src/main/java"/>
+	<classpathentry kind="src" output="build/classes/java/test" path="src/test/java"/>
+	<classpathentry kind="src" output="build/classes/java/testannotations" path="src/testannotations/java"/>
 	<classpathentry kind="src" path="/dataflow"/>
+	<classpathentry kind="src" path="/framework"/>
 	<classpathentry kind="src" path="/javacutil"/>
-	<classpathentry kind="src" path="/checker-qual"/>
+	<classpathentry kind="src" path="/jsr308-langtools"/>
```

Comment:
One more thing: at the moment every project depends on the other projects. It would be nicer if e.g. `checker` didn't need to depend on `jsr308-langtools` and `dataflow` directly, because it got that dependency from `framework`.
In the IDE, in the "Java Build Path" under "Order and Export" you can export the packages and thereby make these dependencies more implicit.